### DISCUSSION TITLE
No more mesh (router to router)

### DISCUSSION THREAD

**Discussion by HA-TB303** [Koenkk/zigbee2mqtt#11221] [General]
Hi, since yesterday my network started behaving weirdly. Routers are no longer connecting to each other and the network map is very weirdly fluctuating.

I have the Slaesh's CC2652RB stick with the latest release firmware, also tried the develop version (CC2652RB_coordinator_20220103.hex)

Zigbee2MQTT version: 1.23.0
Coordinator type: zStack3x0
Coordinator revision: 20220103

Does anyone now how to fix this?

https://user-images.githubusercontent.com/9609805/152514020-43f8ebfb-5c38-48b8-9368-2fd7d0a4b6bb.mp4

<img width="1552" alt="screenshot_ 2022-02-04 11 24 07" src="https://user-images.githubusercontent.com/9609805/152514045-75516dab-7b9f-4eed-ae15-f2d348e2f099.png">

**Comment by HA-TB303** [comment]
I tried everything to fix this, even repairing wouldn't work. I removed zigbee2mqtt, changed the pan id and network key and then re-paired all 75 devices ;)

Everything is working fine now, even re-pairing went so much smoother.

A huge shoutout to all the people who tried to help ;)

Cheers,

Michel

**Comment by AmyYummy** [comment]
Have you tried to find the [manufacturer](www.baudcom.cn) of this router?

**Comment by HA-TB303** [comment]
I have the same issue again. This time after a planned power outage of 2 hours. My network was working absolutely fantastic for nearly 2 weeks, but now again it seems to be the exact same issue.

I'm just about done with this zigbee stuff. Re-pairing 75+ devices is no fun.

Anyone any idea on how to fix this?
